src/js/clipboard.js

```diff
@@ -0,0 +1,640 @@
+                      trigger: this.trigger,
+                      clearSelection: this.clearSelection.bind(this),
+                    });
+                  },
+                },
+                {
+                  key: "clearSelection",
+                  value: function () {
+                    this.trigger && this.trigger.focus(),
+                      window.getSelection().removeAllRanges();
+                  },
+                },
+                {
+                  key: "destroy",
+                  value: function () {
+                    this.removeFake();
+                  },
+                },
+                {
+                  key: "action",
+                  set: function () {
+                    var t =
+                      arguments.length > 0 && void 0 !== arguments[0]
+                        ? arguments[0]
+                        : "copy";
+                    if (
+                      ((this._action = t),
+                      "copy" !== this._action && "cut" !== this._action)
+                    )
+                      throw new Error(
+                        'Invalid "action" value, use either "copy" or "cut"'
+                      );
+                  },
+                  get: function () {
+                    return this._action;
+                  },
+                },
+                {
+                  key: "target",
+                  set: function (t) {
+                    if (void 0 !== t) {
+                      if (
+                        !t ||
+                        "object" !== (void 0 === t ? "undefined" : r(t)) ||
+                        1 !== t.nodeType
+                      )
+                        throw new Error(
+                          'Invalid "target" value, use a valid Element'
+                        );
+                      if ("copy" === this.action && t.hasAttribute("disabled"))
+                        throw new Error(
+                          'Invalid "target" attribute. Please use "readonly" instead of "disabled" attribute'
+                        );
+                      if (
+                        "cut" === this.action &&
+                        (t.hasAttribute("readonly") ||
+                          t.hasAttribute("disabled"))
+                      )
+                        throw new Error(
+                          'Invalid "target" attribute. You can\'t cut text from elements with "readonly" or "disabled" attributes'
+                        );
+                      this._target = t;
+                    }
+                  },
+                  get: function () {
+                    return this._target;
+                  },
+                },
+              ]),
+              t
+            );
+          })();
+        t.exports = a;
+      });
+    },
+    function (t, e, n) {
+      function o(t, e, n) {
+        if (!t && !e && !n) throw new Error("Missing required arguments");
+        if (!c.string(e))
```

Comment:
Codacy found an issue: ['c' was used before it was defined.](https://app.codacy.com/manual/hello_100/jca-ui-kit/pullRequest?prid=5731813)